src/containers/monsters/SingleMonsterContainer.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Monster from '../../components/monsters/Monster';
+
+export default function SingleMonsterContainer(props) {
```

Comment:
it's a container that don't have any sideEffects and dependencies, I don't see any useSelector, useDispatch, async code. So it's just a component